src/kubernetes/kafka/kafka-svc.yaml

```diff
@@ -1,16 +1,16 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: kafka
+  name: kafka-broker
   labels:
-    app: kafka
+    app: kafka-broker
     component: kafka-broker
 spec:
   ports:
   - port: 9092
-    name: kafka-port
+    name: kafka-broker-port
     targetPort: 9092
     protocol: TCP
   selector:
-    app: kafka
-    component: kafka-broker
\ No newline at end of file
+    app: kafka-broker
+    component: kafka-broker-c
```

Comment:
why `c`? anything special about it except the name `Christian` ;)

Reply:
because the component name was already `kafka-broker` when the app name was `kafka`. After we renamed the app to `kafka-broker` i had to change the component name.  In a lack of imaginations i picked `c` (from component) but feel free to rename it.